models/basic-home-example/IHome.json

```diff
@@ -8,6 +8,11 @@
       "@type": "Property",
       "name": "id",
       "schema": "string"      
+    },
+    {
+      "@type": "Telemetry",
+      "name": "Temperature",
+      "schema": "double"      
```

Comment:
i don't know if i would put telemetry property on the Home model. It might get confusing. in this context I wiould want to know the temp of the house and property makes more sense. Having the Telemetry on the sensor would make the most sense